ui/stories/block-error.stories.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/extensions */
+import hbs from 'htmlbars-inline-precompile';
+import { storiesOf } from '@storybook/ember';
+//import notes from './block-error.md';
+
+storiesOf('BlockError/', module)
+  .addParameters({ options: { showPanel: true } })
+  .add(
+    `BlockError`,
+    () => ({
+      template: hbs`
+        <h5 class="title is-5">Block Error</h5>
+        <BlockError/>
```

Comment:
I think we want to have some content in here - currently it's just a grey box.

Reply:
yeah, i removed this component for the time being since it wasn't fleshed out yet. we'll add it in in the next PR!

Reply:
I don't think there's much else to it, it's just a presentational component (and only used in one place) - so not sure we need to include it. All that to say 🎉, sounds good 😄